tests/datasource/fluent/data_asset/test_sql_asset.py

```diff
@@ -97,55 +97,76 @@ def test_add_batch_definition_fluent_sql__add_batch_definition_whole_table(datas
+    datasource, asset, sort_ascending
+):
     # arrange
     name = "batch_def_name"
     column = "test_column"
     expected_batch_definition = BatchDefinition(
-        name=name, partitioner=PartitionerYear(column_name=column), batching_regex=None
+        name=name,
+        partitioner=PartitionerYear(column_name=column, sort_ascending=sort_ascending),
+        batching_regex=None,
     )
     datasource.add_batch_definition.return_value = expected_batch_definition
 
     # act
-    batch_definition = asset.add_batch_definition_yearly(name=name, column=column)
+    batch_definition = asset.add_batch_definition_yearly(
+        name=name, column=column, sort_ascending=sort_ascending
+    )
 
     # assert
     assert batch_definition == expected_batch_definition
     datasource.add_batch_definition.assert_called_once_with(expected_batch_definition)
 
 
 @pytest.mark.unit
-def test_add_batch_definition_fluent_sql__add_batch_definition_monthly(datasource, asset):
+@pytest.mark.parametrize("sort_ascending", (True, False))
+def test_add_batch_definition_fluent_sql__add_batch_definition_monthly(
+    datasource, asset, sort_ascending
+):
     # arrange
     name = "batch_def_name"
     column = "test_column"
     expected_batch_definition = BatchDefinition(
-        name=name, partitioner=PartitionerYearAndMonth(column_name=column), batching_regex=None
+        name=name,
+        partitioner=PartitionerYearAndMonth(column_name=column, sort_ascending=sort_ascending),
+        batching_regex=None,
     )
     datasource.add_batch_definition.return_value = expected_batch_definition
 
     # act
-    batch_definition = asset.add_batch_definition_monthly(name=name, column=column)
+    batch_definition = asset.add_batch_definition_monthly(
+        name=name, column=column, sort_ascending=sort_ascending
+    )
 
     # assert
     assert batch_definition == expected_batch_definition
     datasource.add_batch_definition.assert_called_once_with(expected_batch_definition)
 
 
 @pytest.mark.unit
-def test_add_batch_definition_fluent_sql__add_batch_definition_daily(datasource, asset):
+@pytest.mark.parametrize("sort_ascending", (True, False))
+def test_add_batch_definition_fluent_sql__add_batch_definition_daily(
+    datasource, asset, sort_ascending
+):
     # arrange
     name = "batch_def_name"
     column = "test_column"
     expected_batch_definition = BatchDefinition(
         name=name,
-        partitioner=PartitionerYearAndMonthAndDay(column_name=column),
+        partitioner=PartitionerYearAndMonthAndDay(
+            column_name=column, sort_ascending=sort_ascending
+        ),
         batching_regex=None,
     )
     datasource.add_batch_definition.return_value = expected_batch_definition
 
     # act
-    batch_definition = asset.add_batch_definition_daily(name=name, column=column)
+    batch_definition = asset.add_batch_definition_daily(
+        name=name, column=column, sort_ascending=sort_ascending
+    )
```

Comment:
The output of this test doesn't change based on the new param right? Could we add a test where the order matters?

Reply:
thanks @cdkini! These methods just return a BatchDefinition, not a list of batches; that functionality is tested in `test_get_batch_list_from_batch_request__sort_ascending/descending`.  I created [a followup ticket](https://greatexpectations.atlassian.net/browse/V1-291) to add integration tests around the fluent batch def api.

Reply:
I will add a comment in that ticket that explicitly lists testing sorters as part of the AC.